### ISSUE TITLE
Create "Humans of CircuitVerse" markdown page

### ISSUE THREAD

**Issue by ShreyaPrasad1209** [CircuitVerse/CircuitVerse#2005]
**Is your feature request related to a problem? Please describe.**
 This feature will show significant contributions of individual members in CircuitVerse so as to appreciate the efforts made by them and motivate others to contribute to open source.

**Describe the solution you'd like**
They can be displayed in following manner:

-    Contribution - Person

This allows people to be grouped into a contribution as generally multiple people work to resolve an issue. It should be written in markdown so that it can be readable on GitHub as well.

**Describe alternatives you've considered**
Currently I'm looking into similiar existing pages, will post here once I get relevant references. You are free to come up with your own ideas.

**Additional context**
None

**Are you working on this? (Yes/No)**
Open for everyone

**Comment by harshitaaaaaa** [comment]
Can you assign this issue to me?

**Comment by ShreyaPrasad1209** [comment]
Sure! @satu0king can you assign this issue to her? Seems like I don't have the required permissions.

**Comment by Shivansh2407** [comment]
@harshitaaaaaa you can start working on the issue, You just need to Comment "Working on it" or something Similar to make it clear to the Admins/Maintainers/Other Contributors that you've taken up this issue. Looking forward to your PR 🚀

**Comment by harshitaaaaaa** [comment]
Yeah, I'm working on it!

**Comment by harshitaaaaaa** [comment]
@ShreyaPrasad1209 I have started making the markdown page. Can you have a look at it whether it looks the same as you expected or do I have to make some changes in it. After that I'll proceed further. I am hereby attaching screenshots, you can have a look at them and please tell me if it needs any changes or improvements.
![image](https://user-images.githubusercontent.com/66556662/101668509-a1cc3a00-3a76-11eb-9c73-75219c91ca1d.png)
![image](https://user-images.githubusercontent.com/66556662/101668547-ad1f6580-3a76-11eb-93f7-b2f6729eaa43.png)

**Comment by ShreyaPrasad1209** [comment]
@harshitaaaaaa this looks good. Some suggestions -

- Give user the option to enter their details manually. 
- Right below Humans of CircuitVerse headline, reserve some space for a introduction paragraph. I will give you the content in some time. Add dummy paragraph for now.

**Comment by harshitaaaaaa** [comment]
@ShreyaPrasad1209 Yeah thankyouuuu. I'll do as directed.

**Comment by harshitaaaaaa** [comment]
@ShreyaPrasad1209 I've completed the frontend part. Now I have to make a column in which users can enter their details manually. For that, I need to do the backend part. I have a few queries, please guide me. 
1. Do we have to use ruby on rails for the backend part?
2. How will we get the validation from all the users so that only they have the access to edit/add their information manually.
Please guide me a bit and I'll do that as well.

**Comment by satu0king** [comment]
@harshitaaaaaa looks really good! 

Sorry for the change here, but I think this need to be part of the main website. So can you make it a html page like 
http://circuitverse.org/humans-of-circuitverse ? 

The idea of having a markdown page was it is easier to add the details. But that may not be possible. We can see other alternatives after you make a PR. 

Also can you add optional links like Linkedin, Twitter, Github for each person?

**Comment by harshitaaaaaa** [comment]
@satu0king Yes I can definitely make an HTML page instead of this markdown page and will add all the additional social options.
So in that HTML page, is there any need to give a separate column to each person so that they can manually edit their details?

**Comment by satu0king** [comment]
> @satu0king Yes I can definitely make an HTML page instead of this markdown page and will add all the additional social options.
> So in that HTML page, is there any need to give a separate column to each person so that they can manually edit their details?

You mean separate row? Yeah i think so, unless you have a better idea

**Comment by harshitaaaaaa** [comment]
@ShreyaPrasad1209 You asked me to add a small paragraph after the heading, till now I've added a dummy paragraph. Please provide we with appropriate content so that I can finish making the page.
@satu0king I have converted the markdown into HTML. Do I have to link it with any other webpage?

**Comment by harshitaaaaaa** [comment]
![image](https://user-images.githubusercontent.com/66556662/102060243-f295b700-3e17-11eb-970b-427ca1459004.png)
![image](https://user-images.githubusercontent.com/66556662/102060268-fa555b80-3e17-11eb-8e0b-f3a218ea6e70.png)
@satu0king Please have a look at it. I have made it an HTML page and also added other links. Can I generate a PR if it looks good to you. If you want some other changes and improvements, please let me know.

**Comment by robin025** [comment]
Working On It..!

**Comment by harshitaaaaaa** [comment]
@robin025 I've already done it. Will soon generate a PR.

**Comment by robin025** [comment]
@ShreyaPrasad1209 can I work on it.....?

**Comment by ShreyaPrasad1209** [comment]
@robin025 let's wait for Harshita's PR and then you can work on it in case there's any issue with the PR.

**Comment by satu0king** [comment]
@harshitaaaaaa generally u make a PR and then get a review not in the issue itself.

**Comment by satu0king** [comment]
@harshitaaaaaa don't like the background color.

**Comment by robin025** [comment]
Hi @ShreyaPrasad1209 , thanks for your reply. I was referring to #2005 as you mentioned to create a "Humans of CircuitVerse" markdown page.

So here my suggestion request!
To be able to use a markdown page that will show significant contributions of individual members in CircuitVerse so as to appreciate the efforts made by them ,we can use Jekyll theme which is a static website generator that’s designed for building minimal, static blogs and pages to be hosted on GitHub Pages.
With the help of jekyll ,it strips everything down to the bare minimum, eliminating a lot of complexity
1.We can easily transforms our plain text into static websites pages and blogs.
2.It requires no databases,cms or any updates
3.It supports static HTML, CSS javascript and even Markdown, so with the help of markdown we can easily use all types of labels like you mentioned in #2005
4.What’s best is it goes well with Github-Pages
5.It loads faster

Also to be able to create a CRUD functionality ,i don't think so this can happen in static website but we can create layouts for contributors in markdown, after that contributors just have to edit those markdown files and commit changes.

looking forward to your reply
thanks

**Comment by ShreyaPrasad1209** [comment]
@robin025 this page will be dynamic because users need to have the option to change their content. A basic CRUD application will be enough I think. Jekyll theme sounds good 👍

**Comment by ShreyaPrasad1209** [comment]
I've added good first issue label for a special project and is currently not open for contributions.

**Comment by Kartikay304** [comment]
@satu0king I want to work on this issue. Can you please, assign this issue to me?

**Comment by Sayan-Maity** [comment]
@satu0king , I want to work on this issue, Can you please assign it to me ?

**Comment by PrateekUp** [comment]
Is this issue open?....I would like to work on it.

**Comment by joannasjohnraj** [comment]
Is this issue still open? I would like to work on it.

**Comment by Urstrulysuvangi** [comment]
What's the update on this issue? Has this been resolved yet? If not, I'd like to work on it. Thank You.

**Comment by harshsharma-11** [comment]
@satu0king , @ShreyaPrasad1209  ,@is issue still open should i can work on this???????????

**Comment by nisha-laur** [comment]
@ShreyaPrasad1209 I want to resolve this issue ,please assign it to me .

**Comment by lavi20** [comment]
I want to work on this issue please assign this to me

**Comment by alokartist** [comment]
@ShreyaPrasad1209 
I've reviewed this issue and created a web design using Figma. You can check out the design at [Design link](https://www.figma.com/file/QF8MsUN7FOWHy2E3QDjtUt/Humans-of-CircuitVerse?type=design&node-id=0%3A1&mode=design&t=3lFa6Xp2C4R4lbuI-1). 
I've attached some pages for your reference. Please take a look and let me know if any further improvements are needed. 
Thanks!

![pg1page1 Humans in circuitverse](https://github.com/CircuitVerse/CircuitVerse/assets/120403676/8a4df3c3-a6b5-453f-a83e-4a763123414f)
![pg2page2 Humans in circuitverse](https://github.com/CircuitVerse/CircuitVerse/assets/120403676/bb02997b-7ae5-4b3e-bd14-e6482b22c12b)
![pg3page3 Humans in circuitverse](https://github.com/CircuitVerse/CircuitVerse/assets/120403676/a428fa32-07f2-4c94-ac49-3a51d5f3b98e)

**Comment by hackwhiz-8** [comment]
hey, could you assign this issue to me.
I will add card for each contributor which will be responsive for mobile devices also.
it will take one day to make all the cards

**Comment by rajatk281** [comment]
Hi maintainers 👋  

I’d like to work on this issue if it is still open.  
I’ve gone through the description and understand the goal of highlighting individual contributors’ efforts in a single, easily shareable place while keeping everything GitHub-readable using markdown.

I’m aligned with the idea of a markdown-based page that lists contributors along with their contribution areas (labels) and a short description, entered manually by the contributors themselves. This keeps the workflow simple and consistent while still making the page meaningful and motivating for the community.

If assigned, I’d first propose a clean and reusable markdown structure (with clear sections, labels, and examples) so that future contributors can easily add themselves without confusion.

Please let me know if I can be assigned to this issue or if you’d prefer a brief outline/mock structure before proceeding.

Thanks for your time!

**Comment by rajatk281** [comment]
Hi maintainers 👋  

I’d like to work on this issue if it’s still open.  
I’ve gone through the description and understand the intent of highlighting individual contributors’ efforts in a single place to appreciate their work and motivate others to contribute.

I’m aligned with the markdown-based approach so that the page remains GitHub-readable and easy to maintain. My plan would be to design a clear and consistent markdown structure where contributors can list their names, contribution areas (labels), and a short description, making it simple for future contributors to add themselves.

If assigned, I’d first share a proposed markdown layout with examples to ensure it aligns with the maintainers’ expectations before proceeding further.

Please let me know if I can be assigned to this issue.

Thanks!

**Comment by SAMREEN111-hash** [comment]
Hi maintainers 👋  

I’d like to work on this issue if it’s still available.  
I’ve gone through the description and I understand the goal of creating a markdown-based page to highlight contributors and make it easy for others to add themselves.

Before starting, I plan to first propose a clean and consistent markdown structure (with examples) and share it here for feedback, to ensure it aligns with your expectations.

Please let me know if I can be assigned to this issue.  
Thanks!
